['Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpgcwqs3z0/solution_0.py", line 86, in <module>\n    print(Solution().subsequencesWithMiddleMode(nums = [0,1,2,3,4,5,6,7,8]))\nTypeError: subsequencesWithMiddleMode() got an unexpected keyword argument \'nums\'\n', 'Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmpgcwqs3z0/solution_1.py", line 86, in <module>\n    print(Solution().subsequencesWithMiddleMode(nums = [1,2,2,3,3,3,4,4,4,4,5,5,5,5,5]))\nTypeError: subsequencesWithMiddleMode() got an unexpected keyword argument \'nums\'\n']